src/borg/archiver.py

```diff
@@ -534,6 +534,7 @@ def create_inner(archive, cache, fso):
                     process_file_chunks=cp.process_file_chunks, add_item=archive.add_item,
                     chunker_params=args.chunker_params, show_progress=args.progress)
                 create_inner(archive, cache, fso)
+                logger.error("Repository %s has been created successfully" % archive.name)
         else:
             create_inner(None, None, None)
```

Comment:
in this branch, there will be no output with your modification. but guess there should be.

dry-run is to simulate a backup run without really writing something, the messages can be the same though.

also maybe move it to before the recursion is started and say something like "Creating archive ... in repository ....".
then we will have that output even in case the backup is aborted later. the final success state is available via --show-rc.